frontend/locales/en/strings.json

```diff
@@ -6,6 +6,19 @@
   },
   "cluster": {
     "list": {
+      "dialogs": {
+        "delete": "Are you sure you want to delete cluster"
```

Comment:
todo: when you need to translate a content parametrized (`clusterName` in this case) you can embed the parameter inside the label and then pass the value when translating the content, like this
```
"delete": "Are you sure you want to delete cluster {{clusterName}}?"

//inside the view
{t("cluster.list.dialogs.delete", { clusterName: "cluster name here" })}
```

Reply:
Thanks, I did not know how to handle it otherwise. Going to add it!